README.md

```diff
@@ -1,6 +1,6 @@
 # Hello-Portfolio
 
-> In this project, I will set up a "Hello-Portfolio" repository. My goal here is to create a both mobile and desktop friendly portfolio page. 
+> In this project, I will set up a "Hello-Portfolio" repository. My goal here is to create the first page of a mobile-friendly portfolio page. 
 
```

Comment:
Please add steps to run the project locally in README.

Reply:
I am not quite sure what I should put in the Getting started section, since it is a simple webpage I don't know what I should add to the ReadMe about it. Please enlighten me, thank you.